### PR TITLE
ignorePrefixed Added

### PR DESCRIPTION
**Pull request by ssovit**
will make things easier so underscore prefixed less files won't be compiled when saved.
Useful specially for mixin files like how I am used to organize

### CODE REVIEW

File: smart_less_build.sublime-settings [hunk 1]
```diff
@@ -15,6 +15,8 @@
 		//String "@same_dir"
 	"minify": false,
 		//Boolean minify or not
+  "ignorePrefixed": true,
+    // Boolean underscore  Ignoreprefixed file
```
Comment (thecotne) [inline smart_less_build.sublime-settings:19]: you need to fix indentation

File: smart_less_build.py [hunk 2]
```diff
@@ -101,6 +101,11 @@ def on_post_save(self, view):
 			cmd +=" --compress"
 
 		cmd +=" --no-color"
+		print(main_less)
+		if os.path.basename(main_less).startswith('_') and self.get('ignorePrefixed'):
+			print("Prefixed File so ignored")
```
Comment (thecotne) [inline smart_less_build.py:106]: remove print from line 106 and 104

Comment (ssovit) [inline smart_less_build.py:106]: I am too lazy you know :D

**Comment by thecotne** [comment]
you don't actually need to ignore prefixed files 
just set `main_less` to your main file and when you change any file in `working_dir` your `main_less` will be compiled

**Pull request closed without merging**